src/app.py

```diff
@@ -1257,7 +1257,7 @@ def create_request_headers_for_auth(self, token):
 
     def init_translator(self, token):
         if self.ubkg_instance is None:
-            return self.translator_module.Translator(self.INDICES, self.APP_CLIENT_ID, self.APP_CLIENT_SECRET, token, self.ONTOLOGY_API_APP_SETTING)
+            return self.translator_module.Translator(self.INDICES, self.APP_CLIENT_ID, self.APP_CLIENT_SECRET, token)
```

Comment:
@kburke what I really meant is to replace this `ONTOLOGY_API_APP_SETTING` dict with a `ONTOLOGY_API_URL` and only use the `application_context` from constant. 

SenNet introduced this `ubkg_instance` from the https://github.com/x-atlas-consortia/commons package, which is not a good design and will probably change down the road. So I don't want to broadcast this to HuBMAP.